scripts/beachball/index.ts

```diff
@@ -15,4 +16,24 @@ export const config: BeachballConfig = {
     },
     groups: [getVNextChangelogGroups()],
   },
+  hooks: {
+    postpublish: (packagePath, name, version) => {
+      // Following https://github.com/microsoft/fluentui/pull/20352 all prerelease versions are released with `latest`
+      // Adds a post publish hook to continue to tag prerelease versions with appropriate tags
+      // can be removed once v9 is fully released
+      if (version.indexOf('beta') !== -1 && process.env.RELEASE_VNEXT) {
+        const tag = 'beta';
```

Comment:
We might want to declare it above the `if` and use the value in the `version.includes()`.